app/models/aliquot/deprecated_behaviours.rb

```diff
@@ -24,26 +24,5 @@ def tag
 
     delegate :tags, to: :asset
     deprecate :tags
-
```

Comment:
I'm almost certain this is unused, its definitely not been hit in our logs and grepping only found two uses, both of which appear redundant. (Although the generic name does mean I'm not 100% certain)